frontend/src/components/wallet/PagePortfolio.vue

```diff
@@ -46,58 +46,9 @@
         </template> -->
       </template>
     </TmPage>
+    <page-loading v-if="isNetworkFetching" />
   </div>
 </template>
-
-<style scoped lang="scss">
```

Comment:
Why did you remove this part? It's crash styles for portfolio( when your authorise

Reply:
there were two style part in the file. So I removed one. You can see another style part in the end of the file.